app/assets/v2/js/status.js

```diff
@@ -243,8 +243,9 @@ $(document).ready(function() {
 
   });
 
+  {% if not no_grow %}
```

Comment:
what is no_grow ? where it comes form? 
Can we just remove autogrow I don't sse it working on prod